library/std/src/process.rs

```diff
@@ -1400,6 +1400,90 @@ impl From<fs::File> for Stdio {
     }
 }
 
+#[stable(feature = "stdio_from", since = "1.20.0")]
```

Comment:
nit: I think this should get its own feature gate and version number, even though it's insta-stable, to avoid confusing users.